### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by Leo-MathGuy**
Changes:
 * Formatting
 * Made chatFilter more efficent by adding a lambda expression inside a lambda expression
 * Made datahandler a bit less messy

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,49 +3,56 @@ plugins {
...
     properties {
         property "sonar.projectKey", "Omega-Network_OmegaPlugin_AYYfMoaq2k4vs7aXpSe6"
     }
 }
 
+// Omegahub is a commercial entity now
 group 'com.omegahub'
 version '2.0'
 
+// Must match min game version in plugin.json
```
Comment (Volas171) [inline build.gradle:17]: nope, this crashes when compiling due to the fact anule has only uploaded 141 to jitpack apparently.....

File: build.gradle [hunk 2]
```diff
@@ -13,7 +13,7 @@ group 'com.omegahub'
 version '2.0'
 
 ext {
-    mindustryVersion = 'v141'
+    mindustryVersion = 'v141.2'
```
Comment (Volas171) [inline build.gradle:19]: As mentioned, this turns the server into dumpster fire due to 141.2 not existing

**Review by Volas171 — COMMENTED** [review]
very nice, however it won't compile as is, anule fucked up and you will need to change 141.2 to 141

**Review by Redstonneur1256 — COMMENTED** [review]
`PreparedStatement` calls cannot be chained, this doesn't compile

**Pull request merged**